utilities/openemr-cmd/openemr-cmd

```diff
@@ -19,91 +20,43 @@ check_docker_compose_install_or_not(){
 }
 
 quick_open_a_docker_shell(){
-	docker exec -it $CONTAINER_ID sh
-}
-
-refresh_page_or_iframe() {
-	$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE '/root/devtools build-themes'
-}
-
-check_php_error_logs() {
-	$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE '/root/devtools php-log'
+	CONTAINER_NAME=$1
+	CONTAINER_NAME_CODE=24	
+	if [ $# -eq 1 ]
+	then	
+		docker exec -it $CONTAINER_NAME sh
+	else
+		echo '********************************************************'
+		echo 'Access the docker openemr_openemr_1 by default.' 
```

Comment:
note that this would be more accurate, since the host may not have openemr in a openemr directory:
echo 'Access the docker *_openemr_1 by default.'

Reply:
> note that this would be more accurate, since the host may not have openemr in a openemr directory:
> 
> ```
> echo 'Access the docker *_openemr_1 by default.'
> ```
ok